### PR TITLE
MR-640 Manage main merchant orders

### PR DESCRIPTION
**Pull request by az-joss**
## PR Description

- Ticket: https://spryker.atlassian.net/browse/MR-640

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-order-management-feature-integration.md [hunk 1]
```diff
@@ -57,6 +57,96 @@ Add the following configuration:
...
+    }
+
+    /**
+     * @return \Spryker\Zed\Kernel\Communication\Form\FormTypeInterface
+     */
+    public function getItemFormTypePlugin(): FormTypeInterface
+    {
+        return new ItemFormTypePlugin();
+    }
+}
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-order-management-feature-integration.md:146]: shouldn't be in Set up behavior section?

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-promotions-discounts-feature-integration.md [hunk 2]
```diff
@@ -35,44 +35,18 @@ Make sure that the following modules were installed:
...
-| MerchantOrder.merchantOrderItems | attribute | created | src/Generated/Shared/Transfer/MerchantOrderTransfer |
-
-{% endinfo_block %}
-
-### 3) Zed translations
-
-Generate a new translation cache for Zed:
-
-```bash
-console translator:generate-cache
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-promotions-discounts-feature-integration.md:66]: why removed?

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-promotions-discounts-feature-integration.md [hunk 3]
```diff
@@ -35,44 +35,18 @@ Make sure that the following modules were installed:
...
-### 4) Set up configuration
-
-Change the following the configuration to enable the feature:
+Add the following configuration:
 
 | CONFIGURATION | SPECIFICATION | NAMESPACE |
-| - | - | - |
-| MerchantSalesOrderMerchantUserGuiConfig   ::getMerchantSalesOrderDetailExternalBlocksUrls | Adds Merchant discount separation while viewing Merchant Order. | \Pyz\Zed\MerchantSalesOrderMerchantUserGui\MerchantSalesOrderMerchantUserGuiConfig |
+| ------------- | ------------- | --------- |
+| MerchantSalesOrderMerchantUserGuiConfig  | Introduces list of urls of order detail page configuration. | src/Pyz/Zed/MerchantSalesOrderMerchantUserGui/MerchantSalesOrderMerchantUserGuiConfig.php |
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-promotions-discounts-feature-integration.md:49]: this is description for a method, no for a config

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-promotions-discounts-feature-integration.md [hunk 4]
```diff
@@ -35,44 +35,18 @@ Make sure that the following modules were installed:
...
 
 {% endinfo_block %}
 
-### 2) Set up the transfer objects
+### 2) Set up configuration
 
-Generate transfer changes:
-
-```bash
-console transfer:generate
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-promotions-discounts-feature-integration.md:48]: please leave this section

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-order-management-cart-feature-integration.md [hunk 5]
```diff
@@ -17,15 +17,16 @@ To start feature integration, integrate the required features:
...
-| Cart            | {{page.version}}    | [Cart feature integration](https://github.com/spryker-feature/cart) |
+| Marketplace Order Management | {{page.version}}  | [Marketplace Order Management feature integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/marketplace-order-management-feature-integration.html) |
 
 ### 1) Install the required modules using Composer
 
 Install the required modules:
 
 ```bash
-composer require spryker/cart-note-merchant-sales-order-gui:"^0.1.0" --update-with-dependencies
+composer require spryker-feature/marketplace-cart:"{{page.version}}" --update-with-dependencies
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-order-management-cart-feature-integration.md:29]: shouldn't be renamed accordingly?

**Review by ilyakubanov — COMMENTED** [review]
- is Order Threshold needed in Prerequisites? There is redirect to checkout
-

**Comment by az-joss** [comment]
> * is Order Threshold needed in Prerequisites? There is redirect to checkout

yes, the [OrderThreshold](https://release.spryker.com/features/view/44) feature is not completed and that is why it redirects to checkout where is a note about threshold.

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-promotions-discounts-feature-integration.md (hunk 2 above)
Comment (az-joss) [reply docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-promotions-discounts-feature-integration.md:66]: not removed, just placed in order as in IG guides.

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-order-management-feature-integration.md (hunk 1 above)
Comment (az-joss) [reply docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-order-management-feature-integration.md:146]: put this into own cross feature IG

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-order-management-cart-feature-integration.md (hunk 5 above)
Comment (ilyakubanov) [reply docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-order-management-cart-feature-integration.md:29]: I mean the the IG name

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-promotions-discounts-feature-integration.md (hunk 3 above)
Comment (az-joss) [reply docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-promotions-discounts-feature-integration.md:49]: make sense, added method name to table

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-order-management-cart-feature-integration.md (hunk 5 above)
Comment (az-joss) [reply docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-order-management-cart-feature-integration.md:29]: did not even notice, fixed

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-order-management-feature-integration.md (hunk 1 above)
Comment (az-joss) [reply docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-order-management-feature-integration.md]: ok, reverted.

File: _data/sidebars/marketplace_dev_sidebar.yml [hunk 6]
```diff
@@ -111,18 +111,21 @@ entries:
           - title: Marketplace Merchant Portal Core feature integration
             url: /docs/marketplace/dev/feature-integration-guides/marketplace-merchant-portal-core-feature-integration.html
 
+          - title: Marketplace Cart feature integration
+            url: /docs/marketplace/dev/feature-integration-guides/marketplace-cart-feature-integration.html
+
```
Comment (ilyakubanov) [inline _data/sidebars/marketplace_dev_sidebar.yml:116]: the old one still left

File: _data/sidebars/marketplace_dev_sidebar.yml [hunk 7]
```diff
@@ -132,6 +135,9 @@ entries:
           - title: Marketplace Order Management + Order Threshold feature integration
             url: /docs/marketplace/dev/feature-integration-guides/marketplace-order-management-order-threshold-feature-integration.html
 
+          - title: Marketplace Product + Cart feature integration
+            url: /docs/marketplace/dev/feature-integration-guides/marketplace-product-cart-feature-integration.html
+
```
Comment (ilyakubanov) [inline _data/sidebars/marketplace_dev_sidebar.yml:140]: why to add second time?

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-cart-feature-integration.md [hunk 8]
```diff
@@ -34,31 +35,17 @@ Make sure that the following modules have been installed:
...
-| MerchantOrder.order | attribute | created   | src/Generated/Shared/Transfer/MerchantOrderTransfer |
-
-{% endinfo_block %}
-
-### 3) Set up behavior
-
-Add the following configuration to the project:
+| CONFIGURATION | SPECIFICATION | NAMESPACE |
+| ------------- | ------------- | --------- |
+| MerchantSalesOrderMerchantUserGuiConfig  | Introduces list of urls of order detail page configuration. | src/Pyz/Zed/MerchantSalesOrderMerchantUserGui/MerchantSalesOrderMerchantUserGuiConfig.php |
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-cart-feature-integration.md:48]: the same - config is presented in the table but description for a method

File: _data/sidebars/marketplace_dev_sidebar.yml (hunk 7 above)
Comment (az-joss) [reply _data/sidebars/marketplace_dev_sidebar.yml:143]: looks like merge issue, fixed

File: _data/sidebars/marketplace_dev_sidebar.yml (hunk 6 above)
Comment (az-joss) [reply _data/sidebars/marketplace_dev_sidebar.yml:116]: ah, means link to the origin file name that was renamed - removed.

**Review by gibmirmilch — APPROVED** [review]

**Pull request merged**